### PR TITLE
zh-trans: developing-cloud-controller-manager.md

### PR DESCRIPTION
**Pull request by zgfh**
zh-trans: /docs/tasks/administer-cluster/developing-cloud-controller-manager.md

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+To build an out-of-tree cloud-controller-manager for your cloud, follow these steps:
+-->
+要为您的云构建一个 out-of-tree 云控制器管理器，请按照下列步骤操作：
+
+<!--
+1. Create a go package with an implementation that satisfies [cloudprovider.Interface](https://git.k8s.io/kubernetes/pkg/cloudprovider/cloud.go).
+2. Use [main.go in cloud-controller-manager](https://github.com/kubernetes/kubernetes/blob/master/cmd/cloud-controller-manager/controller-manager.go) from Kubernetes core as a template for your main.go. As mentioned above, the only difference should be the cloud package that will be imported.
+3. Import your cloud package in `main.go`, ensure your package has an `init` block to run [cloudprovider.RegisterCloudProvider](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/plugins.go#L42-L52).
+-->
+1. 使用满足 [cloudprovider.Interface](https://git.k8s.io/kubernetes/pkg/cloudprovider/cloud.go) 的实现创建一个 go 依赖包。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md]:
```suggestion
1. 使用满足 [cloudprovider.Interface](https://git.k8s.io/kubernetes/pkg/cloudprovider/cloud.go) 的实现创建一个 go 包。
```

File: content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+2. 使用来自 Kubernetes 核心包的 [cloud-controller-manager 中的 main.go](https://github.com/kubernetes/kubernetes/blob/master/cmd/cloud-controller-manager/controller-manager.go) 作为 main.go 的模板。如上所述，唯一的区别应该是将导入的云包。
+3. 在 `main.go` 中导入你的云包，确保你的包有一个 `init` 块来运行 cloudprovider.RegisterCloudProvider。
+
+<!--
+Using existing out-of-tree cloud providers as an example may be helpful. You can find the list [here](/docs/tasks/administer-cluster/running-cloud-controller.md#examples).
+-->
+用现有的 out-of-tree 云提供商为例可能会有所帮助。你可以在这里找到 [清单](/docs/tasks/administer-cluster/running-cloud-controller.md#examples)。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md]:
```suggestion
用现有的 out-of-tree 云驱动作为例子可能会有所帮助。你可以在这里找到 [清单](/docs/tasks/administer-cluster/running-cloud-controller.md#examples)。
```

Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md]: 在本文中，provider 均可翻译为驱动

**Comment by zgfh** [comment]
@markthink  @tengqm 感谢，已经修改

**Comment by Rajakavitha1** [comment]
@tengqm Please take a look at this PR and state your suggestions/approval.

**Pull request merged**